src/modules/reportes_poai/reportes_poai.service.ts

```
@@ -0,0 +1,665 @@
+            correo,
+            identificacion
+          ),
+          poai:poai(
+            id,
+            periodo,
+            created_at,
+            updated_at
+          )
+        `);
+
+            // Aplicar filtro de fecha específica
```

ESLint

[error] Delete `······`

(prettier/prettier)